app/components/file/report-btn/template.hbs

```diff
@@ -0,0 +1,137 @@
+<div {{did-insert this.initializeComp}}>
+
+  {{#if @isSecurityDashboard}}
+    <div local-class='file-report-btn' data-test-report='analysis'>
+      <button type='button' class='button is-primary' local-class='btn-container' {{on 'click' this.onGenerateReport}} data-test-report="security-generate-report">
+        {{t 'generateReport'}}
+      </button>
+      {{! Download specific reports like excel, HTML etc,. }}
+      <div class="dropdown dropdown--default" local-class='external-download' data-test-report='external-report-download-dropdown'>
+        <BasicDropdown @renderInPlace={{true}} @horizontalPosition='auto-right' as |dd|>
+          <dd.trigger class='trigger-bootstrap-feel button is-primary' local-class='dd-trigger'>
+            <div local-class='trigger-item' data-test-report='ext-download-trigger'>{{fa-icon 'ellipsis-v'}}</div>
+          </dd.trigger>
+          <dd.content @class='dropdown-content dropdown--default-content'>
+            <div class="dropdown-pointer"></div>
+            <div class="dropdown-container" local-class='dd-content' data-test-report='ext-download-content'>
+              {{#each this.externalReportTypes as |type seq|}}
+                <a href='#' class="dropdown-item" local-class='external-report-item' {{on 'click' (perform this.downloadExternalReport type)}}
+                  data-test-report='external-report-{{seq}}'>
+                  {{fa-icon type.icon}}
+                  {{t 'download'}} {{type.label}}
+                </a>
+              {{/each}}
+            </div>
+          </dd.content>
+        </BasicDropdown>
+      </div>
+    </div>
```

Comment:
Don't mix the security dashboard logic into main dashboard components. That's a bug NO!

Reply:
oh okie, it wasn't exposed anywhere since the beginning of the task so, it's continue to be there. Will move it to separate component then 👍